File: C++/Playing with values/Sum of Digits/sum_digits.cpp
#include<iostream>

using namespace std;

int main ()
{
    int val, num, sum = 0;

    cout << " Enter the number : ";
    cin >> val;
    cout << "\n";

    num = val;

    while (num != 0){
        sum = sum + num % 10;
        num = num / 10;
    }

    cout << " The sum of the digits is = " << sum << endl;

    return 0;
}

File: JAVA/Basic Programs/Sum of Digits/src/sum_digits.java
import java.util.Scanner;

public class sum_digits {
	public static void main (String args[]) {
		int num, og_num, sum = 0;
		
		Scanner inp = new Scanner (System.in);
		
		System.out.print(" Enter the Number : ");
		og_num = inp.nextInt();
		
		num = og_num;
		
		while ( num != 0 ) {
			sum += num % 10;
			num /= 10;
		}
		
		System.out.println(" The Sum of the Digits = " + sum);
	}
}
